distributed/worker.py

```diff
@@ -2625,7 +2625,9 @@ def secede():
     """
     Have this task secede from the worker's thread pool
 
-    This opens up a new scheduling slot and a new thread for a new task.
+    This opens up a new scheduling slot and a new thread for a new task. This
+    enables the client to schedule another task while waiting for the
+    ``client.gather`` to finish.
```

Comment:
The user isn't necessarily using `gather` here.  It might be best to generalize this comment a bit.

Reply:
How does that look? (sorry, it's not showing up in GitHub UI)